{
	"id": "oval:org.opensuse.security:def:20169390",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2016-9390",
		"affected": {
			"family": "unix",
			"platform": [
				"openSUSE Tumbleweed"
			]
		},
		"reference": [
			{
				"ref_id": "Mitre CVE-2016-9390",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-9390",
				"source": "CVE"
			},
			{
				"ref_id": "SUSE CVE-2016-9390",
				"ref_url": "https://www.suse.com/security/cve/CVE-2016-9390",
				"source": "SUSE CVE"
			},
			{
				"ref_id": "SUSE-SU-2017:1901-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2017-July/003040.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2017:1916-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2017-July/003054.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "SUSE-SU-2018:0339-1",
				"ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2018-February/003697.html",
				"source": "SUSE-SU"
			},
			{
				"ref_id": "openSUSE-SU-2017:1960-1",
				"source": "SUSE-SU"
			}
		],
		"description": "\n    The jas_seq2d_create function in jas_seq.c in JasPer before 1.900.14 allows remote attackers to cause a denial of service (assertion failure) via a crafted image file.\n    ",
		"advisory": {
			"from": "security@suse.de",
			"severity": "Moderate",
			"cve": [
				{
					"text": "CVE-2016-9390 at SUSE",
					"href": "https://www.suse.com/security/cve/CVE-2016-9390/",
					"impact": "moderate",
					"cvss3": "5.5/CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
				},
				{
					"text": "CVE-2016-9390 at NVD",
					"href": "https://nvd.nist.gov/vuln/detail/CVE-2016-9390",
					"impact": "moderate",
					"cvss3": "5.5/CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
				}
			],
			"bugzilla": [
				{
					"text": "SUSE bug 1010774",
					"href": "https://bugzilla.suse.com/1010774"
				},
				{
					"text": "SUSE bug 1078851",
					"href": "https://bugzilla.suse.com/1078851"
				},
				{
					"text": "SUSE bug 1178702",
					"href": "https://bugzilla.suse.com/1178702"
				}
			],
			"issued": {
				"date": "2021-10-02"
			},
			"updated": {
				"date": "2024-04-18"
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/o:opensuse:tumbleweed"
				]
			}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:org.opensuse.security:tst:2009638019",
						"comment": "jasper-2.0.33-1.2 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009638020",
						"comment": "libjasper-devel-2.0.33-1.2 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009638021",
						"comment": "libjasper4-2.0.33-1.2 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009638022",
						"comment": "libjasper4-32bit-2.0.33-1.2 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009634834",
				"comment": "openSUSE Tumbleweed is installed"
			}
		]
	}
}
